icons/hdmi-port.svg

```diff
@@ -0,0 +1,14 @@
+<svg
```

Comment:
Feels a bit small and is hard to identify as a port.
What about something like this?
![icon (87)](https://github.com/lucide-icons/lucide/assets/25524993/da3fd233-5024-4657-8bf6-1660d7bdcc3d)